### ISSUE TITLE
[BUG]: Marked tag not applying (or displaying ?) correctly

### ISSUE THREAD

**Issue by AXiSource** [ankidroid/Anki-Android#14228]
### Checked for duplicates?

- [X] This issue is not a duplicate

### What are the steps to reproduce this bug?

1. Mark a card in Ankidroid
2. Unknown
3. Discrepancy across instances.

Important : Note that if I tag "marked" a single card then sync immediatly it works correctly.

### Expected behaviour

Same notes being tagged "marked" across multiple instances.

### Actual behaviour

"marked" discrepancy between multiples instances of Anki.
Everything is synced and the content of the notes is syncing correctly across instances too.

<details>
<summary>Ankidroid 2.16.2 on Samsung Android 13 - My Main Device</summary>

![WhatsApp Image 2023-08-14 at 09 38 02](https://github.com/ankidroid/Anki-Android/assets/32098707/bee98a5c-9a83-4402-b85d-d6e9dc1a9f15)
</details>

_________

<details>
<summary>Ankidroid 2.16.2 on Android 9</summary>

![WhatsApp Image 2023-08-14 at 09 55 17](https://github.com/ankidroid/Anki-Android/assets/32098707/164611de-f6fb-4215-8d54-0c418a46e70e)
</details>

________

<details>
<summary>Anki Desktop 2.1.65</summary>

![Screenshot (1)](https://github.com/ankidroid/Anki-Android/assets/32098707/86c368da-afba-4b3b-9a06-15f2da848508)
</details>

________

Can somebody tell me how to do collapsable pictures please ?

### Debug info

```text
AnkiDroid Version = 2.16.2

Android Version = 13

ProductFlavor = play

Manufacturer = samsung

Model = SM-M135F

Hardware = exynos850

Webview User Agent = Mozilla/5.0 (Linux; Android 13; SM-M135F Build/TP1A.220624.014; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/115.0.5790.166 Mobile Safari/537.36

ACRA UUID = 7ab61a33-0623-4db8-9604-6b458ef84023

New schema = false

Scheduler = std2

Crash Reports Enabled = true

DatabaseV2 Enabled = true
```
**DEBUG INFO FOR 2ND PHONE :**

```text
AnkiDroid Version = 2.16.2

Android Version = 9

ProductFlavor = play

Manufacturer = samsung

Model = SM-J530F

Hardware = samsungexynos7870

Webview User Agent = Mozilla/5.0 (Linux; Android 9; SM-J530F Build/PPR1.180610.011; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/115.0.5790.166 Mobile Safari/537.36

ACRA UUID = a905acb9-550b-4875-98d9-ac8fda4c2ac7

New schema = false

Scheduler = std2

Crash Reports Enabled = true

DatabaseV2 Enabled = true
```

### Research

- [X] I am reporting a bug specific to AnkiDroid (Android app)
- [X] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [X] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by david-allison** [comment]
Hi, thanks for the report. Could you provide debug information for both phones?

**Comment by AXiSource** [comment]
Updated Debug Info.

Sorry, I already removed all the marked "tag" on my main phone "by hand".
In my learning process, I mark cards with my phone, treat them with my PC, then sync them back to my phone.
The marked cards change everyday.
If the bug comes back I will upload cards.

Many thanks for your answer and your time.

**Comment by david-allison** [comment]
Should be fixed in 2.17, let me know if this still occurs
